src/dpq2/conv/from_d_types.d

```diff
@@ -20,14 +20,27 @@ import vibe.data.json: Json;
 
 /// Converts Nullable!T to Value
 Value toValue(T)(T v)
-if (is(T == Nullable!R, R))
+if (is(T == Nullable!R, R) && !(isArrayType!(typeof(v.get))))
 {
     if (v.isNull)
         return Value(ValueFormat.BINARY, detectOidTypeFromNative!T);
     else
         return toValue(v.get);
 }
 
+/// ditto
+Value toValue(T)(T v)
+if (is(T == Nullable!R, R) && (isArrayType!(typeof(v.get))))
+{
+    import dpq2.conv.arrays : arrToValue = toValue; // deprecation import workaround
```

Comment:
Sucessfully tried to remove this line with DMD64 D Compiler v2.082.1 (at least, CI was passed)